modules/ossm-about-accessing-bookinfo-application-using-gateway.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-accessing-bookinfo-application-using-gateway"]
+= About accessing the Bookinfo application using a gateway
+:context: ossm-about-accessing-bookinfo-application-using-gateway
+
+The {SMProductName} Operator does not deploy gateways. Gateways are not part of the control plane. As a security best-practice, Ingress and Egress Gateways should be deployed in a different namespace than the namespace that contains the control plane.
```

Comment:
Ingress and Egress Gateways

maybe lowercase  for  gateways

Reply:
done